### PR TITLE
[WIP] Adding new test of service retire manual approval

### PR DESCRIPTION
**Pull request by niyazRedhat**
{{ pytest: cfme/tests/services/test_service_manual_approval.py -k 'test_service_retire_manual_approval' --long-running --use-template-cache -sqvvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 2e10c65ed90c7dab3c4f4a00e36e0bc590fc57d1

The *local* fixture **`retirement_domain`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *retirement_instance*

The *local* fixture **`retirement_instance`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *test_service_retire_manual_approval*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 1b4956bbe571a281096b72845a3f940cc2d1f836

The *local* fixture **`retirement_domain`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *retirement_instance*

The *local* fixture **`retirement_instance`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *test_service_retire_manual_approval*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/services/test_service_manual_approval.py [hunk 1]
```diff
@@ -26,6 +29,36 @@
 ]
 
 
+@pytest.fixture(scope="function")
+def retirement_domain(request, appliance):
+    """Create new domain and copy instance from ManageIQ to this domain"""
+
+    dc = DomainCollection(appliance)
+    new_domain = dc.create(name=fauxfactory.gen_alphanumeric(), enabled=True)
+    request.addfinalizer(new_domain.delete_if_exists)
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: Please use global fixture to create automate domain.
```def retirement_domain(request, appliance, domain):```

File: cfme/tests/services/test_service_manual_approval.py [hunk 2]
```diff
@@ -26,6 +29,36 @@
...
+    """Modify the instance in new domain to change it to manual approval instead of auto"""
+
+    instance = (retirement_domain.namespaces.instantiate(name='Service')
+        .namespaces.instantiate(name='Retirement')
+        .namespaces.instantiate(name='StateMachines')
+        .classes.instantiate(name='ServiceRetirementRequestApproval')
+        .instances.instantiate(name='Default'))
+    with update(instance):
+        instance.fields = {"approval_type": {"value": "manual"}}
+
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: Can we merge these two fixtures: ```retirement_domain``` and ```retirement_instance```?

File: cfme/tests/services/test_service_manual_approval.py [hunk 3]
```diff
@@ -87,3 +120,51 @@ def test_service_manual_approval(appliance, provider, modify_instance,
                                                                  partial_check=True)
     service_request.update(method='ui')
     assert service_request.row.approval_state.text == 'Pending Approval'
+
+
+@pytest.mark.meta(automates=[BZ(1697600)])
+@pytest.mark.customer_scenario
+def test_service_retire_manual_approval(appliance, service_vm, retirement_instance):
+    """" Test service retirement manual approval
+
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: ```"""" Test service retirement manual approval``` (Remove extra **"**)

Comment (ganeshhubale) [inline cfme/tests/services/test_service_manual_approval.py]: Add Bugzilla in doc string.

**Comment by niyazRedhat** [comment]
closing this . raised new pr with new changes

**Pull request closed without merging**